front/app/containers/Admin/reporting/components/ReportBuilder/Editor/RenderNode/index.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { useEffect } from 'react';
+import styled from 'styled-components';
+
+// components
+import { Box } from '@citizenlab/cl2-component-library';
+
+// styles
+import { colors } from 'utils/styleUtils';
+
+// craft
+import { useNode, useEditor, ROOT_NODE } from '@craftjs/core';
+
+// intl
+import { FormattedMessage, MessageDescriptor } from 'utils/cl-intl';
+import messages from 'components/admin/ContentBuilder/Editor/RenderNode/messages';
+
+const StyledBox = styled(Box)`
+  ${({ isRoot }: { isRoot: boolean }) =>
+    isRoot
+      ? `cursor: auto;
+padding-top: 4px;
+padding-bottom: 4px;
+width: 100%;
+max-width: 1000px;
+background-color: #fff;
+min-height: 160px;`
+      : `cursor:move;`}
+`;
+
+const CONTAINER = 'Container';
+
+const RenderNode = ({ render }) => {
```

Comment:
Had to create our own `RenderNode` because the existing one was adding some extra padding to the template, which made everything look off when trying to add other things to the report outside of the template